akd_mysql/src/mysql_storables.rs

```diff
@@ -138,7 +138,7 @@ impl MySqlStorable for DbRecord {
                 "hash" => node.latest_node.hash,
                 // "Previous" node values
                 "p_last_epoch" => node.previous_node.clone().map(|a| a.last_epoch),
-                "p_least_descendant_ep" => node.previous_node.clone().map(|a| a.least_descendant_ep),
+                "p_least_descendant_ep" => node.previous_node.clone().map(|a| a.smallest_descendant_ep),
```

Comment:
Ah OK so this is "technically" a contract break, since it's something MySQL sees. Therefore can you bump the minimum version of `akd` that `akd_mysql` depends on? Additionally we should version-bump all crates with this, since it's not backwards compatible.

Reply:
Bumped everything to 0.8.1